### PR TITLE
【M4】Manage Group

### PR DESCRIPTION
**Pull request by kanatagu**
# Description
* Added Manage Group page

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Check List
* You can change permission if you're the owner of the trip
* You can delete the user if you're the owner of the trip
* Otherwise, you can not change and delete the users

## URL
http://localhost:3000/trip/[tripId]/manage-group

## Screenshot
PC
screencapture-localhost-3000-trip-63248b67-37a8-42bd-abaa-cd07f158974b-manage-group-2024-02-18-19_32_46

screencapture-localhost-3000-trip-63248b67-37a8-42bd-abaa-cd07f158974b-manage-group-2024-02-18-19_32_50

SP

* if you're not the owner, the radio button is just READ ONLY and you can not see the delete button
Screenshot 2024-02-18 at 7 35 10 PM

VIDEO

https://github.com/tabi-memo/frontend/assets/66394413/90b4cec2-c36e-4184-b9ba-276cb606f97e

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -5,8 +5,8 @@
     "**/.hg": true,
     "**/CVS": true,
     "**/.DS_Store": true,
-    "node_modules": true,
-    ".next": true
+    "node_modules": false,
+    ".next": false
```
Comment (kanatagu) [inline .vscode/settings.json:9]: Sometimes, we want to delete cache and node_modules manually, so I changed this place 🙏

File: app/trip/[id]/page.tsx [hunk 2]
```diff
@@ -76,14 +87,7 @@ export default function TripDetailsPage({
...
               costUnit={tripData.edges[0].node.cost_unit}
-              users={
-                tripData.edges[0].node.invitationsCollection?.edges.map(
-                  (invitation) => ({
-                    id: invitation.node.users?.id,
-                    image: invitation.node.users?.profile_picture_url
-                  })
-                ) || []
-              }
+              users={[owner, ...sharedUsers]}
```
Comment (kanatagu) [inline app/trip/[id]/page.tsx:56]: I realised that we only showed invitation users except the owner, so fixed it

File: app/trip/components/trip-card.tsx [hunk 3]
```diff
@@ -83,29 +83,47 @@ export const TripCard = ({ data }: TripCardProps) => {
...
+                        objectFit="cover"
+                      />
+                    ) : (
+                      <MdAccountCircle size="100%" />
+                    )}
+                  </Box>
+                ))}
+              </>
+            )}
+        </Flex>
```
Comment (kanatagu) [inline app/trip/components/trip-card.tsx:126]: This is the same. We want to show the owner too!

File: app/trip/hooks/useTripsGet.ts [hunk 4]
```diff
@@ -5,6 +5,7 @@ import { useTripsCollectionQuery, TripsOrderBy } from '@generated/api'
 export const useTripsGet = (searchWord?: string) => {
   const userId = useUserId()
 
+  // TODO - need to change as currently just showing all trips user created. We also need to show invited trips
```
Comment (kanatagu) [inline app/trip/hooks/useTripsGet.ts:8]: This is the one I asked on Slack.
It's complicated to fix it so I created the issue : https://github.com/tabi-memo/frontend/issues/37

**Review by samuraikun — APPROVED** [review]
LGTM :star:

**Pull request merged**